### PR TITLE
Update keras.io guide on KerasCV Object detection for v0.4.1

### PR DESCRIPTION
**Pull request by LukeWood**

### CODE REVIEW

**Comment by LukeWood** [comment]
Hmm, it looks like the inference was broken at some point.  will fix first - then we can review.

File: guides/keras_cv/retina_net_overview.py
Comment (fchollet) [inline guides/keras_cv/retina_net_overview.py:295]: So should we remove this?

**Review by fchollet — APPROVED** [review]
LGTM -- should we merge or do we need an additional review from Ian or Zhenyu?

**Comment by LukeWood** [comment]
> LGTM -- should we merge or do we need an additional review from Ian or Zhenyu?

Lets keep lightweight for now and we can revisit later when more updates are made (i.e. AutoAnchor, inference updates).

Let me re-generate the genfiles before merging, I will tell you when.

**Comment by LukeWood** [comment]
ok @fchollet feel free to merge, this is more updated than the original.  We will need to continue tuning this guide with time - but the current one is entirely broken so this is a net positive.

**Review by fchollet — APPROVED** [review]
LGTM, thanks!

**Pull request merged**
